static/js/util/test_utils.js

```diff
@@ -1,7 +1,36 @@
 import TestUtils from 'react-addons-test-utils';
+import sinon from 'sinon';
 
 export const modifyTextField = (field, text) => {
   field.value = text;
   TestUtils.Simulate.change(field);
   TestUtils.Simulate.keyDown(field, {key: "Enter", keyCode: 13, which: 13});
 };
+
+export const localStorageMock = (init = {}) => {
+  let storage = init;
+
+  const getItem = sinon.spy(key => storage[key] || null);
+
+  const setItem = sinon.spy((key, value) => {
+    storage[key] = value || "";
+  });
+
+  const removeItem = sinon.spy(key => {
+    delete storage[key];
+  });
+
+  const reset = () => {
+    [getItem, setItem, removeItem].forEach(func => {
```

Comment:
Can you give an example of how I could make that work? I tried playing around with a sandbox and I couldn't get it to play nicely with the spy functions.